RecordHub.Shared/MassTransit/Models/Order/BasketCheckoutMessage.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace RecordHub.Shared.MassTransit.Models.Order
+{
+    public class BasketCheckoutMessage : IntegrationBaseEvent
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string EmailAddress { get; set; }
+        public string UserId { get; set; }
+        public IEnumerable<OrderItemModel> Items { get; set; }
```

Comment:
move collection to the bottom

Reply:
in dev its already in the bottom, i can change it here too or when merging pr keep dev's

Reply:
Ok, don't fix here, if it have already done in your main